internal/dag/gatewayapi_processor.go

```diff
@@ -119,7 +120,7 @@ func (p *GatewayAPIProcessor) Run(dag *DAG, source *KubernetesCache) {
 
 			// ## RouteBindingSelector ##
 			//
-			// Selector specifies a set of route labels used for selecting routes to associate
+			// Selector specifies a set of hostnames labels used for selecting routes to associate
```

Comment:
looks like there are a bunch more find/replace errors in this and other file, I'm not gonna comment on them all, hopefully you can go through and revert all the erroneous ones.